pkgs/build-support/setup-hooks/update-golang-sys-hook.sh

```diff
@@ -0,0 +1,14 @@
+preConfigureHooks+=(_updateGolangSys)
+
+_updateGolangSys() {
+    sed -i '/golang.org\/x\/sys/d' go.mod
+    cat >>go.mod <<EOF
+require (
+    golang.org/x/sys v0.0.0-20220731174439-a90be440212d // indirect
+)
+EOF
+    cat >>go.sum <<EOF
```

Comment:
Is this always working? go can be very nity about its vendoring files. What about modules.txt?

Reply:
Indeed, there are packages on which the hook doesn't work, e.g. `podgrab`, with the error

```
golang.org/x/sys@v0.0.0-20220731174439-a90be440212d: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
```

Otherwise, the hook already works pretty well on most failing packages on Hydra.